### ISSUE TITLE
C/octane: not serializing an object in JSON test

### ISSUE THREAD

**Issue by michaelhixson** [TechEmpower/FrameworkBenchmarks#4094]
C/octane's implementation of the JSON test is non-compliant because it doesn't serialize an object to JSON.

https://github.com/TechEmpower/FrameworkBenchmarks/blob/ec98e030f36b3a4bbae01d66a89de7626429a977/frameworks/C/octane/src/responders/sds_responder.cpp#L28-L33

I don't know what the best way to make it compliant is.  If we can't make it compliant, we should remove the JSON URL from its `benchmark_config.json` file, which would effectively disable the test.

**Comment by zloster** [comment]
@michaelhixson IMO this could be closed now.

I've tried to fix C/octane's other problems mentioned in the RP but my C knowledge is limited and rusty.

I've copied the `libreactor` date header formatting implementation:
https://github.com/fredrikwidlund/libreactor/blob/46b244594e752863aa793931724d1964edf7daaf/src/reactor/reactor_http.c#L553-L565
But the verification fails to pass and gives me the error `The framework is not responding to requests from the client machine`.

The `.hpp, .h, .cpp, .c` part is also tricky for me because the used JSON serialisation library is C++-based. This seems to affect how the code is compiled. I couldn't make it work with just renaming the files.
